mono/metadata/class-accessors.c

```diff
@@ -6,11 +6,16 @@
 #include <mono/metadata/tabledefs.h>
 
 
-enum InfrequentDataKind {
+typedef enum {
 	PROP_MARSHAL_INFO = 1, /* MonoMarshalType */
-	PROP_EXT = 2, /* MonoClassExt */
-	PROP_REF_INFO_HANDLE = 3, /* gchandle */
-};
+	PROP_REF_INFO_HANDLE = 2, /* gchandle */
```

Comment:
One thing to keep in mind is that the bigger the value, the more expensive it is to fetch it.

We should take a look on what properties are more popular or used during steady state and prioritize those.